api-samples/declarativeNetRequest/no-cookies/README.md

```diff
@@ -0,0 +1,11 @@
+# chrome.declarativeNetRequest - No Cookies
+
+This sample demonstrates how to use the [chrome.declarativeNetRequest](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/) API to remove the "Cookie" header from requests.
```

Comment:
>This sample demonstrates how to use the

For your consideration: this is an example of an inflated verb. Either 'this sample demonstrates using' or 'this sample uses' would get the point across. (The first example is not an inflated verb because 'using' is the start of a gerund: a verb functioning as a noun.)